utils/nctl/sh/assets/upgrade.sh

```diff
@@ -180,7 +179,7 @@ function _generate_global_state_update_balances() {
     # First, we supply the path to the directory of the node whose global state we'll use
     # and the trusted hash.
     local PARAMS
-    PARAMS="balances -d ${STATE_SOURCE_PATH}/storage/$(get_chain_name) -s ${STATE_HASH} -f ${SRC_ACC} -t ${TARGET_ACC} -a ${AMOUNT} -p ${PROPOSER}"
+    PARAMS="balances -d ${STATE_SOURCE_PATH}/storage/$(get_chain_name) -s ${STATE_HASH} -f ${SRC_ACC} -t ${TARGET_ACC} -a ${AMOUNT}"
```

Comment:
What did `-p` do originally?

Reply:
Originally this command worked by actually executing a transfer on the execution engine. And when a transfer is executed, the user initiating it has to pay a fee for the execution, which goes to the proposer of the block. So `-p` was specifying who should be considered the proposer.

Now it will work by just directly modifying the balances in the global state, which means no fee for execution and no need to specify the proposer.